.travis.yml

```diff
@@ -3,14 +3,8 @@ language: python
 
 matrix:
   include:
-    - python: 2.7
-      env: TOXENV=flake8
```

Comment:
@mirii1994 Can we run flake8 with a newer python version instead of deleting it?

Reply:
Checked it now, it's possible. Committing changes.